src/style.css

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+}
+
+.container {
```

Comment:
the container is redundant just style header, and use paddings instead of margins